sdk/purview/Azure.Analytics.Purview.Sharing/src/autorest.md

```diff
@@ -0,0 +1,59 @@
+# Azure SDK Code Generation for Data Plane
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+input-file: 
+- https://github.com/Azure/azure-rest-api-specs/blob/12cbd66750c4dfe302e9eca42b1fc727f8c02700/specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2023-02-15-preview/share.json
+namespace: Azure.Analytics.Purview.Sharing
```

Comment:
@m-nash could you please take a cursory glance at autorest settings and other generated files abide by our latest recommendation.